core/scripts/chaincli/handler/keeper.go

```diff
@@ -596,3 +661,42 @@ func (k *Keeper) getConfigForRegistry12() *registry12.Config {
 		Registrar:            common.HexToAddress(k.cfg.Registrar),
 	}
 }
+
+func (k *Keeper) changeDirectory(directory string) (string, error) {
```

Comment:
I wonder could we simplify this function, so that we:

- Get the current directory
- Construct the absolute path
- Instead of checking if we are already in the current directory, always change directory to the absolute path
- Check if the config file exists, and if not, return an error

So, rather than checking if we are in the directory already, we just cd into the absPath anyway?

Reply:
"checking if we are already in the current directory" is needed because we could verify multiple contracts in a run, in our case, we verify 3. So we dont want to change the directory all the time, hence checking if it is already in target directory, if so, dont move. I will refactor the code a bit to make the logic cleaner.

Reply:
+1 to what ferglor said